.gitignore

```diff
@@ -30,8 +23,8 @@ nuget.exe
 *.ncrunchsolution
 *.*sdf
 *.ipch
-.settings
 *.sln.ide
-node_modules
-**/[Cc]ompiler/[Rr]esources/**/*.js
-*launchSettings.json
```

Comment:
Pretty sure we added `launchSettings.json` on purpose because it started showing up in this repo.